movement/io/save_poses.py

```diff
@@ -104,9 +186,50 @@ def to_dlc_file(ds: xr.Dataset, file_path: Union[str, Path]) -> None:
         logger.error(error)
         raise error
 
-    df = to_dlc_df(ds)  # convert to pandas DataFrame
-    if file.path.suffix == ".csv":
-        df.to_csv(file.path, sep=",")
-    else:  # file.path.suffix == ".h5"
-        df.to_hdf(file.path, key="df_with_missing")
-    logger.info(f"Saved PoseTracks dataset to {file.path}.")
+    # Sets default behaviour for the function
+    if split_individuals is None:
+        individuals = ds.coords["individuals"].data.tolist()
+        if len(individuals) == 1:
+            split_individuals = True
+        else:
+            split_individuals = False
+
+    """If split_individuals is True then it will split the file into a
+    dictionary of pandas dataframes for each individual."""
+    if split_individuals:
```

Comment:
Again here, as in the above function, we have to check if there are more than one individuals to split, otherwise output only one single-animal file.

Reply:
If split_individuals is True and there is only one individual to split it should already automatically output only one single-animal file.